pkg/fanal/cache/fs_test.go

```diff
@@ -341,6 +359,7 @@ func TestFSCache_PutArtifact(t *testing.T) {
 				    {
 				      "Name": "musl",
 				      "Version": "1.2.3",
+					  "Identifier": {},
```

Comment:
We need to add logic for `apk` from imgConf - https://github.com/aquasecurity/trivy/tree/main/pkg/fanal/analyzer/imgconf/apk

Let me time think about this.

Reply:
done at https://github.com/aquasecurity/trivy/pull/5439/commits/f96248bed003c2526e644eb89cc369bf3e4f0279